applications/targets/tests/test_target_create.py

```diff
@@ -87,3 +87,33 @@ def test_no_loged_user_respond_unauthorized(self):
         response = self.call_create_target()
 
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_created_10_targets_responds_failure(self):
+        targets_count = 10
```

Comment:
The "10" in the name is kind of a magic number. Maybe it would be better something referring to the maximum allowed. Continuing with the previous suggestion, you could import the `MAX_COUNT_TARGETS_PER_USER` and use it here.